data/CMakeLists.txt

```diff
@@ -17,4 +17,4 @@ install(FILES
   panel
   start
   vars
-  DESTINATION etc/pekwm/)
+  DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/pekwm/)
```

Comment:
OK.. changing this to

 DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/pekwm/)

makes it possible install in the right places with just `-DCMAKE_INSTALL_PREFIX=/usr`

Reply:
ok, let me test locally

Reply:
This change will divert from the default location to /usr/... and we would need to put override   -DCMAKE_INSTALL_SYSCONFDIR=/usr/etc to comply with the default location.
Thing is if I don't spesify anything this change conforms to my distro default /usr/bin and /etc/pekwm, so this is fine by me, but if Alpine or any other distros shove everything under /usr/... then this change probably won't be accepted.

Reply:
That's exactly the intent. See https://cmake.org/cmake/help/v3.5/module/GNUInstallDirs.html#special-cases. People who want to put sysconfdir elsewhere needs to specify both, but most distros should only need install prefix.

Reply:
@pclouds sorry I misspelled you name in commit message.

Reply:
Not at all a problem.